src/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <PropertyGroup>
+    <VersionPrefix>1.2.2</VersionPrefix>
+    <VersionSuffix>dev</VersionSuffix>
```

Comment:
Can you explain what the resulting version string is going to be? What happens when we want to publish a release?

Reply:
`1.2.2-dev` when built manually, `1.2.2-ci.{build id}` when built by CI, `1.2.2` when built by CI with a tag (after I figure out how to do it with appveyor)

Reply:
> when built by CI with a tag (after I figure out how to do it with appveyor)
Should happen asap after